### DISCUSSION TITLE
no color display

### DISCUSSION THREAD

**Discussion by HaoRan-hash** [cnr-isti-vclab/meshlab#1099] [Q&A]
I use MeshLab 2021.07, platform is windows 10.
I import mesh [file](https://github.com/HaoRan-hash/meshlab_use/blob/main/chair_2.txt).
options are as follows.
![屏幕截图 2021-09-26 160444](https://user-images.githubusercontent.com/54461374/134799243-c956beeb-3443-4203-938a-17b79d728dd5.jpg)

The result is below.
![屏幕截图 2021-09-26 160658](https://user-images.githubusercontent.com/54461374/134799409-21cf9bf9-cdfc-47bc-aa80-49044f6b7d6e.jpg)
**no color display**, I do not konw the reason, hope someone can help me.

**Comment by alemuntoni** [comment]
Hi, it seems to me that there is vertex color, it's just a bit dark..
It becomes more evident making the size of the vertices bigger:
![image](https://user-images.githubusercontent.com/6656168/134917015-c73fc8dc-eb11-4a87-b8eb-ef321d60eb3d.png)
Also, in the violet bar at the bottom of the drawing area, the "VC" line means that the mesh displayed has vertex color!
